apps/expo/src/app/contact-us/index.tsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react'
+import { Linking, Text, View } from 'react-native'
+
+import { Feather, Fontisto, SimpleLineIcons } from '@expo/vector-icons'
+
+import NavBar from '~/components/ui/nav-bar/NavBar'
+
+export default function ContactUsPage() {
+  return (
+    <View style={{ flex: 1, backgroundColor: '#1C1B1B' }}>
+      {/* Header */}
```

Comment:
You could try adding a safe view or something along those lines to add some padding to the top of the screen (react-native component, `import {SafeAreaView} from 'react-native'`.

Reply:
I agree, but don't use `SafeAreaView` from the `react-native` package. Use the one from the longer named import (I don't remember what it's called).

Reply:
@StefanoRubini RE: #4 in your PR update

You didn't read my message :'( Using the `SafeAreaView` from the `'react-native'` import doesn't work. It needs to be imported from `'react-native-safe-area-context'`:

`import { SafeAreaView } from 'react-native-safe-area-context'`

Reply:
Hello Elliot, I will make sure to incorporate the necessary changes into my next commit, thank you for bringing up this issue! 😃

Reply:
@StefanoRubini no need to add padding to the top of the page. This is taken care of in the NavBar component in #249 . Just add the SafeAreaView that Elliot mentioned.